plugin/napari_lattice/_reader.py

```diff
@@ -138,3 +141,24 @@ def bdv_h5_reader(path):
 
     layer_type = "image"  # optional, default is "image"
     return [(images, add_kwargs, layer_type)]
+
+
+def tiff_reader(path):
+    """Take path to tiff image and returns a list of LayerData tuples.
+    Specifying tiff_reader to have better control over tifffile related errors when using AICSImage
+    """
+    
+    try:
+        image = AICSImage(path)
+    except Exception:
+        import sys
+        exception = sys.exc_info()[0]
+        print("Error Reading Tiff: ", sys.exc_info()[
+                0], "has occurred. Try upgrading tifffile library: pip install tifffile --upgrade.")
+        raise
+
+    # optional kwargs for the corresponding viewer.add_* method
+    add_kwargs = {}
+
+    layer_type = "image"  # optional, default is "image"
+    return [(image, add_kwargs, layer_type)]

```

Comment:
```suggestion
    return image, add_kwargs, layer_type
```
I don't think this needs to be a list? This syntax will return a tuple.